### ISSUE TITLE
Change width of the Textfield

### ISSUE THREAD

**Issue by patreu22** [n4kz/react-native-material-textfield#26]
Hello,

thx very much for that nice component.
Now I'm looking for how to change that component's width. 
Did I miss something?

**Comment by n4kz** [comment]
Hi,

`Style` prop is passed down to underlying `TextInput` component, so you have to use a wrapper view with width restriction.

```javascript
<View style={{ width: 200 }}>
  <TextField />
</View>
```

**Comment by patreu22** [comment]
Great, thank you very much for the quick answer :)

**Comment by n4kz** [comment]
Added `containerStyle` prop to pass styles directly to container view. Released in `0.6.4`.

Now it's possible to set width without a wrapper

```javascript
<TextField containerStyle={{ width: 200 }} />
```
